README.md

```diff
@@ -1,16 +1,16 @@
 # Connect
 
-The "Connect" component is responsible for the communication with a target system.
-This component implements the logic of the interaction protocol, receiving and sending messages from and to the system, respectively.
+The "Connect" component is responsible for the communication with the target system.
+This component implements the logic of the interaction protocol, by receiving and sending messages from and to the system respectively.
 
-This project includes only an adapter for using the Sailfish service in the th2 packed into a Docker Image.
-This image should be used as a base to implement extensions with the real logic for specific protocols using services in the Sailfish format.
+This project includes only one adapter for using the Sailfish service in the th2 package into the Docker Image.
```

Comment:
This sentence has completely changed its meaning.
This project is the adapter that is packed into the Docker Image. This adapter allows users to use the Sailfish services in the th2